models/model.py

```diff
@@ -0,0 +1,368 @@
+#room base
+class Room():
+    '''
+    models abstract data type Room, not intended to be instanciated
+    '''
+    #keep track of all rooms created
+    number_of_rooms = 0
+
+    def __init__(self, max_occupants, name):
+        '''
+        return Room with no occupats
+        '''
+        Room.number_of_rooms += 1
```

Comment:
This statement might not be necessary, you can check the number of occupants in a room. instead of incrementing the rooms class.